docs/site/development/cldr-big-red-switch/generating-charts.md

```diff
@@ -2,27 +2,26 @@
 title: Generating Charts
 ---
 
-# Generating Charts
-
 ## Generate
 
-The input for this is the cldr\-staging/production file, and the output is in github cldr\-staging. **(If for the development version, the input is main or the maint branch, but we should change that.)**
+The input for this process is the `../cldr-staging/production` file, and the output is in the github repo [`cldr-staging`](https://github.com/unicode-org/cldr-staging). **(If for the development version, the input is `main` or the `maint` branch, but we should change that.)**   `cldr-staging` is expected to be checked out as a _sibling_ directory to your CLDR source repository, hence `../cldr-staging`.
```

Comment:
Does GenerateAllCharts really read from cldr-staging?

Reply:
also I wonder if we need a separate page discussing production data and cldr-staging - nothing else on the site about it now